### PR TITLE
[Doc Improvement][Share to Teams support in Gov clouds][3777626]

### PR DESCRIPTION
**Pull request by v-preethah**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/share-to-teams-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/da407eb3abcdd5cf79327bc0075f392fe217b8a5/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshare-to-teams-overview.md) | 88 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/43c95fe8-e3d8-456c-949d-199348aa422d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/share-to-teams-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d69bf8857960f07e4d6dcffd8c607052d81fc24e/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshare-to-teams-overview.md) | 91 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3df255a2-c041-46bf-986b-8a6f9cc179d9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/share-to-teams-overview.md [hunk 1]
```diff
@@ -10,11 +10,14 @@ ms.date: 02/09/2023
...
 
+> [!NOTE]
+> Share to Teams isn't available for [Government Community Cloud (GCC) High and Department of Defense (DOD)](~/concepts/app-fundamentals-overview.md#government-community-cloud) environments.
+
 You can also decide the placement of the Share to Teams button on the app. Users can pin the crucial content and refer to it while collaborating with Teams.
 
 :::image type="content" source="../../assets/images/share-to-teams/overview.PNG" alt-text="Screenshot showing the Share button on the app.":::
 
-You can enable this feature on the following:
+You can enable this feature on the following scopes:
```
Comment (v-npaladugu) [inline msteams-platform/concepts/build-and-test/share-to-teams-overview.md:20]:
```suggestion
You can enable Share to Teams button in the following:
```

File: msteams-platform/concepts/build-and-test/share-to-teams-overview.md [hunk 2]
```diff
@@ -10,11 +10,14 @@ ms.date: 02/09/2023
...
 
 * Web apps
 * Personal app or tab
```
Comment (v-npaladugu) [inline msteams-platform/concepts/build-and-test/share-to-teams-overview.md:23]: Add the gov clouds note in Ln# 24, after the bullet points.

File: msteams-platform/concepts/build-and-test/share-to-teams-overview.md (hunk 1 above)
Comment (v-preethah) [reply msteams-platform/concepts/build-and-test/share-to-teams-overview.md:20]: Updating as:

You can enable Share to Teams button in the following scopes:

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/share-to-teams-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/48e21d97f28f8972067b7b33a93dc65975e12b8d/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshare-to-teams-overview.md) | 91 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/84ded0a5-c65e-4959-bc2f-1a50a50f312f) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/concepts/build-and-test/share-to-teams-overview.md [hunk 3]
```diff
@@ -10,11 +10,14 @@ ms.date: 02/09/2023
 
 Share to Teams is a button enabled on any web app, personal app, or tab that allows users to share content to other users in Teams.
 
+> [!NOTE]
+> Share to Teams isn't available for [Government Community Cloud (GCC) High and Department of Defense (DOD)](~/concepts/app-fundamentals-overview.md#government-community-cloud) environments.
```
[the remark this answers is not in the document]
Comment (v-preethah) [reply msteams-platform/concepts/build-and-test/share-to-teams-overview.md]: Retaining

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/share-to-teams-overview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/48e21d97f28f8972067b7b33a93dc65975e12b8d/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fshare-to-teams-overview.md) | 91 | 2 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/964dd133-7363-4431-9536-5f1e60a4978c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-npaladugu — APPROVED** [review]

**Review by aaronmah — APPROVED** [review]

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**